Allow using the 64-bit build for all boards

farosas opened the pull request:
Our 64-bit build should be able to run the 32 bit machines so add an
option to use qemu-system-ppc64 for all the tests.

Signed-off-by: Fabiano Rosas <farosas@linux.ibm.com>

legoater commented:
I would do this a bit differently.

First, I would introduce at the top of the script a `qemu64=${qemu}64` variable and replace in the machine test definitions `qemu64=64` by `qemu=$qemu64`. That's a good independent cleanup.

Then, to force the run with a 64-bit QEMU, I would simply assign `qemu` to `$qemu64` for all tests. 

Maybe use `--64` for the option name, there wouldn't be a short option in that case but I think it is better than `-s`.

legoater commented:
> First, I would introduce at the top of the script a `qemu64=${qemu}64` variable and replace in the machine test definitions `qemu64=64` by `qemu=$qemu64`. That's a good independent cleanup.

I just did that part.

farosas commented:
> Maybe use `--64` for the option name, there wouldn't be a short option in that case but I think it is better than `-s`.

I agree, I'll change that.

Thanks!

farosas committed:
Allow using the 64-bit build for all boards

Our 64-bit build should be able to run the 32 bit machines so add an
option to use qemu-system-ppc64 for all the tests.

Signed-off-by: Fabiano Rosas <farosas@linux.ibm.com>
